web-ui/src/common/header/header.spec.js

```diff
@@ -12,6 +12,6 @@ describe('Header', () => {
   });
 
   it('renders the header content', () => {
-    expect(header.find('header').text()).toContain('logout');
+    expect(header.find('header').text()).toContain('Logout');
```

Comment:
Is this supposed to test the text or that the component exists? If its the component, then we should use `header.find('header').find('Logout')`. If it's the text, then is this translated?